packages/block-library/src/audio/save.js

```diff
@@ -8,7 +8,9 @@ export default function save( { attributes } ) {
 
 	return (
 		<figure>
-			<audio controls="controls" src={ src } autoPlay={ autoplay } loop={ loop } preload={ preload } />
+			{ src && (
```

Comment:
wonder fi we should just return `null` for the whole block if there's no src 🤷‍♂

Reply:
If nothing at all was shown, then the caption (if any) provided by the user would also not be shown. I'm not sure whether or not that makes sense. Perhaps it would make the most sense to show some text saying something like "missing audio file"?

Reply:
Hi @ZebulanStanphill,
The user cannot provide a caption if an audio file is not selected so I guess we could follow @youknowriad idea.